src/Native/Unix/CMakeLists.txt

```diff
@@ -267,3 +267,5 @@ endif()
 if(APPLE)
     add_subdirectory(System.Security.Cryptography.Native.Apple)
 endif()
+
+add_subdirectory(../AnyOS/SleepTestAppNative build)
```

Comment:
Should /AnyOS/ have its own CMakeLists.txt. (I am not sure how CMake finds these and it's all wired up but maybe that is cleaner than the Unix and Windows ones having to both be updated when adding something in AnyOS?

Reply:
When there will be multiple entrypoints to cmake project, then in case of Windows, we would need to have another `vcxproj` for install step: https://github.com/dotnet/corefx/blob/724d8f7/src/Native/build-native.cmd#L136. It is bit more complicated than the current approach, which is not too much different than what has been done for brotli and zlib under AnyOS.

I think if we want AnyOS to have its own CMakeLists.txt, we can have a platform-agnostic `CMakeLists.txt` at the root of `src/Native` first, to keep one single cmake entrypoint, which can conditionally include Unix and Windows directory based on host platform. This will also make it aligned with CoreCLR: https://github.com/dotnet/coreclr/blob/ca96ed41a6/CMakeLists.txt#L168.